lightning/src/ln/channel.rs

```diff
@@ -2167,8 +2167,8 @@ impl<Signer: Sign> Channel<Signer> {
 	// Get the fee cost in SATS of a commitment tx with a given number of HTLC outputs.
 	// Note that num_htlcs should not include dust HTLCs.
 	#[inline]
-	fn commit_tx_fee_sat(feerate_per_kw: u32, num_htlcs: usize) -> u64 {
-		feerate_per_kw as u64 * (COMMITMENT_TX_BASE_WEIGHT + num_htlcs as u64 * COMMITMENT_TX_WEIGHT_PER_HTLC) / 1000
+	fn commit_tx_fee_sat(feerate_per_kw: u32, num_htlcs: usize, opt_anchors: bool) -> u64 {
+		feerate_per_kw as u64 * (if opt_anchors { 1124 } else { COMMITMENT_TX_BASE_WEIGHT} + num_htlcs as u64 * COMMITMENT_TX_WEIGHT_PER_HTLC) / 1000
```

Comment:
can this commit be squashed away?

Reply:
actually, looks like it's in the final diff.  it should be a constant instead of a magic number?

Reply:
fixed by finishing the COMMITMENT_TX_BASE_WEIGHT conversion